src/Glpi/Dashboard/Widget.php

```diff
@@ -349,9 +349,17 @@ public static function bigNumber(array $params = []): string
             : "";
 
         $label = \htmlescape($p['label']);
-        $alt = \htmlescape($p['alt']);
         $icon = \htmlescape($p['icon']);
 
+        if (empty($p['alt'])) {
+            $p['alt'] = sprintf(
+                __('%1$s %2$s'),
+                $p['number'],
+                $label,
+            );
+        }
```

Comment:
This should be move above the `$label = \htmlescape($p['label']);` line to prevent double escaping issues.

Reply:
Fixed in last commit